sdk/web-pubsub/web-pubsub-client/src/webPubSubClient.ts

```diff
@@ -332,20 +336,39 @@ export class WebPubSubClient {
   }
 
   /**
-   * Send custom event to server
+   * Send custom event to server fire and forget
+   * @param eventName - The event name
+   * @param content - The data content
+   * @param dataType - The data type
+   * @param options - The options
+   * @param abortSignal - The abort signal
+   */
+  public async sendEvent(
+    eventName: string,
+    content: JSONTypes | ArrayBuffer,
+    dataType: WebPubSubDataType,
+    options: SendEventOptionsFireAndForget
+  ): Promise<void>;
+  /**
+   * Send custom event to server with ack
    * @param eventName - The event name
    * @param content - The data content
    * @param dataType - The data type
-   * @param ackId - The optional ackId. If not specified, client will generate one.
    * @param options - The options
    * @param abortSignal - The abort signal
    */
+  public async sendEvent(
+    eventName: string,
+    content: JSONTypes | ArrayBuffer,
+    dataType: WebPubSubDataType,
+    options?: SendEventOptionsAsync
+  ): Promise<WebPubSubResult>;
   public async sendEvent(
     eventName: string,
     content: JSONTypes | ArrayBuffer,
     dataType: WebPubSubDataType,
     options?: SendEventOptions
```

Comment:
Here you should be able to do 

```suggestion
    options?: SendEventOptionsAsync | SendEventOptionsFireAndForget
```

And remove the current SendEventOptions and rename SendEventOptionsAsync => SendEventOptions

Reply:
Updated